client/signup/steps/site-information/index.jsx

```diff
@@ -164,8 +164,8 @@ class SiteInformation extends Component {
 	render() {
 		const { flowName, positionInFlow, signupProgress, stepName, translate } = this.props;
 
-		const headerText = translate( 'Almost done, just a few more things.' );
-		const subHeaderText = translate( "We'll add this information to your new website." );
+		const headerText = translate( 'Help customers find you' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Help your customers find you with Jetpack.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).